modules/extensions-xml-support/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you got the right feeling.
I'm fixing the issues you pointed out and I'll build one